### DISCUSSION TITLE
How to draw a circle

### DISCUSSION THREAD

**Discussion by mrQuidome** [liodali/osm_flutter#301] [Q&A]
I try to draw a circle on the map and don't understand where to put the code in my app. I can follow the example of course. First of all it is an async function so that one won't fit into my widget constructor or it does but then I don't understand how. 
Is there a working example for drawing a circle?

**Comment by liodali** [comment]
why you need widget constructor to draw circle ? 
check example home_example.dart to understand more how to use the library

**Comment by mrQuidome** [comment]
OK, got it working. Also multiple circles :)
